lab_3_generate_by_ngrams/main.py

```diff
@@ -104,6 +141,12 @@ def _put(self, element: str) -> None:
         In case of corrupt input arguments or invalid argument length,
         an element is not added to storage
         """
+        if not isinstance(element, str) or len(element) != 1:
+            return None
+        if len(self._storage) == 0:
```

Comment:
Empty sequences are mapped to False anyway

Reply:
And, by the way, according to the task it is impossible

Reply:
Done

Reply:
is it?